R/dfm.R

```diff
@@ -226,14 +226,21 @@ dfm.tokens <- function(x,
                           x = 1L,
                           dims = c(length(x),
                                    length(type)))
-    dfm.dfm(
-        build_dfm(
-            temp, type,
-            docvars = get_docvars(x, user = TRUE, system = TRUE),
-            meta = attrs[["meta"]]),
-        tolower = FALSE, verbose = verbose
-    )
+
+    temp <- build_dfm(
+        temp, type,
+        docvars = get_docvars(x, user = TRUE, system = TRUE),
+        meta = attrs[["meta"]])
+    
+    if (attrs$meta$object$what != "dictionary") {
```

Comment:
This is to make the order of dictionary keys and column identical. This also skip tokens objects that returned by `tokens_lookup(exclusive = FALSE)`. One solution is not setting what = "dictionary" in non-exclusive mode.